### PR TITLE
new icon: yogabytedb (original, plain, original-wordmark, plain-wordmark)

### PR DESCRIPTION
**Pull request by AbdelrahmanDwedar**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

- https://www.yugabyte.com/

### CODE REVIEW

File: icons/yogabytedb/yogabytedb-plain.svg
Comment (Gautam25Raj) [inline icons/yogabytedb/yogabytedb-plain.svg:1]:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M34.343 5.968c2.939 0 4.375 0 5.51.568a5.324 5.324 0 0 1 2.272 2.271c.568 1.102.568 2.572.568 5.511v36.706c0 4.342 0 6.546-.902 7.882a5.214 5.214 0 0 1-3.44 2.205c-1.604.267-3.574-.668-7.515-2.505l-10.287-4.81c-7.415-3.473-11.122-5.21-13.861-7.882a22.433 22.433 0 0 1-5.377-8.417C.04 33.924.04 29.85.04 21.633v-7.315c0-2.94 0-4.375.568-5.51A5.324 5.324 0 0 1 2.88 6.535c1.136-.568 2.606-.568 5.511-.568Zm85.335 0c2.94 0 4.376 0 5.511.568a5.324 5.324 0 0 1 2.271 2.271c.568 1.102.568 2.572.568 5.511v7.315c0 8.182 0 12.29-1.27 15.898a22.433 22.433 0 0 1-5.376 8.416c-2.706 2.672-6.447 4.409-13.861 7.883L85.344 64.15V14.318c0-2.94 0-4.375.567-5.51a5.324 5.324 0 0 1 2.272-2.272c1.135-.568 2.605-.568 5.544-.568ZM85.344 64.15v49.832c0 2.94 0 4.375-.568 5.51a5.324 5.324 0 0 1-2.271 2.272c-1.103.568-2.572.568-5.511.568H51.042c-2.939 0-4.375 0-5.51-.568a5.324 5.324 0 0 1-2.272-2.271c-.567-1.102-.567-2.572-.567-5.511v-7.348c0-8.183 0-12.29 1.269-15.898a22.433 22.433 0 0 1 5.377-8.417c2.705-2.672 6.446-4.408 13.86-7.849z"/></svg>
```

File: icons/yogabytedb/yogabytedb-plain-wordmark.svg
Comment (Gautam25Raj) [inline icons/yogabytedb/yogabytedb-plain-wordmark.svg:1]:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M125.741 61.54c.814-.421 1.593-1.352 1.593-2.612 0-2.16-1.71-3.366-4.116-3.366h-5.105v13.156h5.286c2.764 0 4.596-1.44 4.596-3.905 0-1.744-1.019-2.822-2.254-3.273zm-5.164-3.998h2.43c1.353 0 2.014.568 2.014 1.592s-.661 1.563-1.984 1.563h-2.465v-3.155Zm2.553 9.01h-2.553V62.8h2.553c1.68 0 2.494.69 2.494 1.891 0 1.206-.814 1.862-2.494 1.862zm-13.724-10.99h-4.262v13.156h4.262c3.665 0 6.757-2.19 6.757-6.575.005-4.39-3.092-6.58-6.757-6.58Zm-.176 10.99h-1.592v-8.858h1.592c2.254 0 4.414 1.17 4.414 4.444 0 3.273-2.165 4.414-4.414 4.414zm-8.109-.99c-.24.902-1.019 1.382-2.224 1.382-1.44 0-2.435-.96-2.612-2.489h6.997c.04-.26.059-.52.059-.779 0-2.283-1.44-4.596-4.625-4.596s-4.655 2.372-4.655 4.866c0 2.493 1.74 4.924 4.836 4.924 2.342 0 4.115-1.289 4.473-3.303h-2.249zm-2.405-4.742c1.381 0 2.19.901 2.312 2.102h-4.684c.24-1.411 1.083-2.102 2.372-2.102zm-5.434.357v-1.95h-2.611v-2.856h-2.313v2.851h-1.862v1.95h1.862v4.234c0 2.102 1.352 3.302 3.425 3.302h1.504v-1.95h-1.26c-.842 0-1.352-.48-1.352-1.44v-4.146h2.607zm-9.672-1.95-2.16 6.517-2.22-6.517h-2.342l3.454 9.49-1.323 3.842h2.19l1.47-4.297 3.303-9.04H83.61zm-11.47-.152c-1.23 0-2.253.54-2.792 1.323v-4.83h-2.313v13.15h2.313v-1.171c.539.813 1.592 1.323 2.734 1.323 2.851 0 4.537-1.98 4.537-4.895-.005-2.92-1.686-4.9-4.479-4.9zm-.391 7.81c-1.534 0-2.553-1.17-2.553-2.915 0-1.744 1.019-2.88 2.553-2.88 1.533 0 2.552 1.14 2.552 2.88 0 1.744-1.019 2.915-2.552 2.915zm-10.873-7.81c-2.464 0-4.051 1.47-4.174 3.366h2.073c.029-.842.779-1.621 2.013-1.621 1.23 0 1.92.661 1.92 1.592 0 .363-.21.632-.69.632h-1.744c-2.131 0-3.572 1.142-3.572 3.003 0 1.622 1.289 2.823 3.15 2.823 1.382 0 2.46-.632 2.852-1.563v1.41h2.283v-6.094c.005-2.136-1.705-3.548-4.11-3.548zm1.833 5.65c0 1.41-.838 2.43-2.283 2.43-.872 0-1.47-.48-1.47-1.23 0-.808.661-1.352 1.622-1.352h2.13zm-21.44-5.498v5.194c0 1.382-.931 2.4-2.225 2.4-1.293 0-1.98-.9-1.98-2.19v-5.404h-2.282v5.885c0 2.22 1.322 3.753 3.513 3.753 1.352 0 2.43-.569 2.974-1.622v1.47h2.312v-9.486Zm-10.064 0-2.19 6.517-2.19-6.517h-2.342l3.454 9.49-1.323 3.842h2.19l1.47-4.297 3.302-9.04h-2.371zm21.293 0v1.083c-.539-.72-1.563-1.23-2.763-1.23-2.822 0-4.473 2.014-4.473 4.895 0 2.88 1.65 4.895 4.502 4.895 1.382 0 2.284-.48 2.764-1.2v.151c0 1.774-.569 2.974-2.43 2.974-1.171 0-1.774-.48-2.161-1.41h-2.313c.51 2.312 2.073 3.33 4.473 3.33 3.004 0 4.626-1.802 4.626-4.894l.088-8.589zm-2.4 7.658c-1.534 0-2.553-1.17-2.553-2.915 0-1.744 1.019-2.88 2.552-2.88 1.534 0 2.583 1.14 2.583 2.88s-1.02 2.915-2.583 2.915zM5.14 55.562c.43 0 .641 0 .808.084a.77.77 0 0 1 .333.333c.083.162.083.377.083.808v5.385c0 .637 0 .96-.132 1.156a.765.765 0 0 1-.505.324c-.235.039-.524-.098-1.102-.368l-1.51-.705c-1.087-.51-1.63-.765-2.032-1.157a3.29 3.29 0 0 1-.79-1.234c-.185-.525-.185-1.122-.185-2.328v-1.073c0-.43 0-.641.083-.808a.781.781 0 0 1 .333-.333c.167-.084.382-.084.808-.084Zm12.518 0c.43 0 .642 0 .808.084a.77.77 0 0 1 .333.333c.084.162.084.377.084.808v1.073c0 1.2 0 1.803-.187 2.333a3.29 3.29 0 0 1-.788 1.234c-.397.392-.946.647-2.034 1.157l-3.253 1.513v-7.31c0-.43 0-.641.083-.808a.781.781 0 0 1 .333-.333c.167-.084.383-.084.814-.084Z"/><path d="M12.621 64.097v7.31c0 .432 0 .642-.083.809a.781.781 0 0 1-.333.333c-.162.083-.378.083-.809.083H7.59c-.431 0-.642 0-.808-.083a.781.781 0 0 1-.333-.333c-.084-.162-.084-.377-.084-.808V70.33c0-1.2 0-1.803.186-2.332a3.29 3.29 0 0 1 .79-1.235c.396-.392.945-.647 2.033-1.152z"/></svg>
```

File: icons/yogabytedb/yogabytedb-original.svg
Comment (Gautam25Raj) [inline icons/yogabytedb/yogabytedb-original.svg:1]:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M34.343 5.968c2.939 0 4.375 0 5.51.568a5.324 5.324 0 0 1 2.272 2.271c.568 1.102.568 2.572.568 5.511v36.706c0 4.342 0 6.546-.902 7.882a5.214 5.214 0 0 1-3.44 2.205c-1.604.267-3.574-.668-7.515-2.505l-10.287-4.81c-7.415-3.473-11.122-5.21-13.861-7.882a22.433 22.433 0 0 1-5.377-8.417C.04 33.924.04 29.85.04 21.633v-7.315c0-2.94 0-4.375.568-5.51A5.324 5.324 0 0 1 2.88 6.535c1.136-.568 2.606-.568 5.511-.568Zm85.335 0c2.94 0 4.376 0 5.511.568a5.324 5.324 0 0 1 2.271 2.271c.568 1.102.568 2.572.568 5.511v7.315c0 8.182 0 12.29-1.27 15.898a22.433 22.433 0 0 1-5.376 8.416c-2.706 2.672-6.447 4.409-13.861 7.883L85.344 64.15V14.318c0-2.94 0-4.375.567-5.51a5.324 5.324 0 0 1 2.272-2.272c1.135-.568 2.605-.568 5.544-.568ZM85.344 64.15v49.832c0 2.94 0 4.375-.568 5.51a5.324 5.324 0 0 1-2.271 2.272c-1.103.568-2.572.568-5.511.568H51.042c-2.939 0-4.375 0-5.51-.568a5.324 5.324 0 0 1-2.272-2.271c-.567-1.102-.567-2.572-.567-5.511v-7.348c0-8.183 0-12.29 1.269-15.898a22.433 22.433 0 0 1 5.377-8.417c2.705-2.672 6.446-4.408 13.86-7.849z" fill="#ff5f3b"/></svg>
```

File: icons/yogabytedb/yogabytedb-original-wordmark.svg
Comment (Gautam25Raj) [inline icons/yogabytedb/yogabytedb-original-wordmark.svg:1]:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M125.741 61.54c.814-.421 1.593-1.352 1.593-2.612 0-2.16-1.71-3.366-4.116-3.366h-5.105v13.156h5.286c2.764 0 4.596-1.44 4.596-3.905 0-1.744-1.019-2.822-2.254-3.273zm-5.164-3.998h2.43c1.353 0 2.014.568 2.014 1.592s-.661 1.563-1.984 1.563h-2.465v-3.155Zm2.553 9.01h-2.553V62.8h2.553c1.68 0 2.494.69 2.494 1.891 0 1.206-.814 1.862-2.494 1.862zm-13.724-10.99h-4.262v13.156h4.262c3.665 0 6.757-2.19 6.757-6.575.005-4.39-3.092-6.58-6.757-6.58Zm-.176 10.99h-1.592v-8.858h1.592c2.254 0 4.414 1.17 4.414 4.444 0 3.273-2.165 4.414-4.414 4.414zm-8.109-.99c-.24.902-1.019 1.382-2.224 1.382-1.44 0-2.435-.96-2.612-2.489h6.997c.04-.26.059-.52.059-.779 0-2.283-1.44-4.596-4.625-4.596s-4.655 2.372-4.655 4.866c0 2.493 1.74 4.924 4.836 4.924 2.342 0 4.115-1.289 4.473-3.303h-2.249zm-2.405-4.742c1.381 0 2.19.901 2.312 2.102h-4.684c.24-1.411 1.083-2.102 2.372-2.102zm-5.434.357v-1.95h-2.611v-2.856h-2.313v2.851h-1.862v1.95h1.862v4.234c0 2.102 1.352 3.302 3.425 3.302h1.504v-1.95h-1.26c-.842 0-1.352-.48-1.352-1.44v-4.146h2.607zm-9.672-1.95-2.16 6.517-2.22-6.517h-2.342l3.454 9.49-1.323 3.842h2.19l1.47-4.297 3.303-9.04H83.61zm-11.47-.152c-1.23 0-2.253.54-2.792 1.323v-4.83h-2.313v13.15h2.313v-1.171c.539.813 1.592 1.323 2.734 1.323 2.851 0 4.537-1.98 4.537-4.895-.005-2.92-1.686-4.9-4.479-4.9zm-.391 7.81c-1.534 0-2.553-1.17-2.553-2.915 0-1.744 1.019-2.88 2.553-2.88 1.533 0 2.552 1.14 2.552 2.88 0 1.744-1.019 2.915-2.552 2.915zm-10.873-7.81c-2.464 0-4.051 1.47-4.174 3.366h2.073c.029-.842.779-1.621 2.013-1.621 1.23 0 1.92.661 1.92 1.592 0 .363-.21.632-.69.632h-1.744c-2.131 0-3.572 1.142-3.572 3.003 0 1.622 1.289 2.823 3.15 2.823 1.382 0 2.46-.632 2.852-1.563v1.41h2.283v-6.094c.005-2.136-1.705-3.548-4.11-3.548zm1.833 5.65c0 1.41-.838 2.43-2.283 2.43-.872 0-1.47-.48-1.47-1.23 0-.808.661-1.352 1.622-1.352h2.13zm-21.44-5.498v5.194c0 1.382-.931 2.4-2.225 2.4-1.293 0-1.98-.9-1.98-2.19v-5.404h-2.282v5.885c0 2.22 1.322 3.753 3.513 3.753 1.352 0 2.43-.569 2.974-1.622v1.47h2.312v-9.486Zm-10.064 0-2.19 6.517-2.19-6.517h-2.342l3.454 9.49-1.323 3.842h2.19l1.47-4.297 3.302-9.04h-2.371zm21.293 0v1.083c-.539-.72-1.563-1.23-2.763-1.23-2.822 0-4.473 2.014-4.473 4.895 0 2.88 1.65 4.895 4.502 4.895 1.382 0 2.284-.48 2.764-1.2v.151c0 1.774-.569 2.974-2.43 2.974-1.171 0-1.774-.48-2.161-1.41h-2.313c.51 2.312 2.073 3.33 4.473 3.33 3.004 0 4.626-1.802 4.626-4.894l.088-8.589zm-2.4 7.658c-1.534 0-2.553-1.17-2.553-2.915 0-1.744 1.019-2.88 2.552-2.88 1.534 0 2.583 1.14 2.583 2.88s-1.02 2.915-2.583 2.915z" fill="#1e154b"/><g fill="#ff5f3b"><path d="M5.14 55.562c.43 0 .641 0 .808.084a.77.77 0 0 1 .333.333c.083.162.083.377.083.808v5.385c0 .637 0 .96-.132 1.156a.765.765 0 0 1-.505.324c-.235.039-.524-.098-1.102-.368l-1.51-.705c-1.087-.51-1.63-.765-2.032-1.157a3.29 3.29 0 0 1-.79-1.234c-.185-.525-.185-1.122-.185-2.328v-1.073c0-.43 0-.641.083-.808a.781.781 0 0 1 .333-.333c.167-.084.382-.084.808-.084Zm12.518 0c.43 0 .642 0 .808.084a.77.77 0 0 1 .333.333c.084.162.084.377.084.808v1.073c0 1.2 0 1.803-.187 2.333a3.29 3.29 0 0 1-.788 1.234c-.397.392-.946.647-2.034 1.157l-3.253 1.513v-7.31c0-.43 0-.641.083-.808a.781.781 0 0 1 .333-.333c.167-.084.383-.084.814-.084Z"/><path d="M12.621 64.097v7.31c0 .432 0 .642-.083.809a.781.781 0 0 1-.333.333c-.162.083-.378.083-.809.083H7.59c-.431 0-.642 0-.808-.083a.781.781 0 0 1-.333-.333c-.084-.162-.084-.377-.084-.808V70.33c0-1.2 0-1.803.186-2.332a3.29 3.29 0 0 1 .79-1.235c.396-.392.945-.647 2.033-1.152z"/></g></svg>
```

File: devicon.json
Comment (Gautam25Raj) [inline devicon.json:1]: **Change Need:**
- Improve alignment
- Add plain, plain-wordmark in SVG versions
- Remove aliases

It seems like you have removed many **devicon** from **devicon.json**. Check them and add them back.

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Hello there! 👋🏼
Please base your pull request (and any future one) on `develop` branch.
Take a look at this comment of how to do it: https://github.com/devicons/devicon/pull/1779#issuecomment-1572094950

Wish for the best! 🚀

**Pull request closed without merging**